Formula/datree.rb

```diff
@@ -6,6 +6,11 @@ class Datree < Formula
   license "Apache-2.0"
   head "https://github.com/datreeio/datree.git", branch: "staging"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

The standard regex for Git tags like `1.2.3`/`v1.2.3` successfully returns the latest version (`0.12.1`), so it doesn't appear to be necessary to use the `GithubLatest` strategy here. We only use `GithubLatest` when it's both necessary and correct.

Reply:
That is right, yeah, I should use regex earlier, good call!